File: rudder-android-client/rudder-sdk-sample-java/src/main/java/com/rudderlabs/android/sdk/MainActivity.java
package com.rudderlabs.android.sdk;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import com.rudderlabs.android.sdk.core.RudderException;
import com.rudderlabs.android.sdk.core.event.*;

public class MainActivity extends AppCompatActivity {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        try {
            RudderElement trackEvent = new RudderElementBuilder().setEventName("Test Track")
                    .setProperty(new TrackPropertyBuilder().setCategory("Test Category")).build();
            MainApplication.getInstance().getRudderClient().track(trackEvent);

            RudderElement pageEvent = new RudderElementBuilder().setEventName("Test Track")
                    .setProperty(new PagePropertyBuilder().setUrl("Test Url")).build();
            MainApplication.getInstance().getRudderClient().page(pageEvent);

            RudderElement screenEvent = new RudderElementBuilder().setEventName("Test Track")
                    .setProperty(new ScreenPropertyBuilder().setScreenName("Test Name")).build();
            MainApplication.getInstance().getRudderClient().screen(screenEvent);

            MainApplication.getInstance().getRudderClient().flush();
        } catch (RudderException e) {
            e.printStackTrace();
        }
    }
}

File: rudder-unity-plugin/android/rudder-sdk-sample-kotlin/src/main/java/com/rudderlabs/android/sample/kotlin/MainActivity.kt
package com.rudderlabs.android.sample.kotlin

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.rudderlabs.android.sdk.core.RudderElementBuilder
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    private var count = 0
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btn.setOnClickListener {
            MainApplication.rudderClient.track(
                "Simple Kotlin Track with '",
                "test_user_id",
                "{\"category\":\"PlayerShooting_Shoot\",\"label\":null,\"value\":null}",
                "null"
            )
            count += 1
            textView.text = "Count: $count"
        }

        rst.setOnClickListener {
            count = 0
            textView.text = "Count: "
        }
    }
}
